SingularityUI/app/templates/vex/requestScale.hbs

```diff
@@ -1,2 +1,13 @@
 <p>Enter the desired number of instances to run for request:</p>
 <pre>{{ id }}</pre>
+
+<label for="bounce" id="bounce-label">
+    <input type="checkbox" id="bounce"{{#if bounceAfterScale}} checked{{/if}}>
+    Bounce after scaling
+</label>
```

Comment:
bonus points for indenting the incremental bounce option + greying it out if the bounce checkmark is unchecked